charts/oes/values.yaml

```diff
@@ -711,6 +703,17 @@ kibana:
 # Spinnaker instance configuration
 #####################################################
 spinnaker:
+  installOpenLdap: true
+  ldap:
```

Comment:
LDAP configuration is already there at line no. 668. To be able to use LDAP config everywhere move it under global